charts/metallb/charts/crds/templates/crds.yaml

```diff
@@ -3,6 +3,9 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.14.0
+    {{- with .Values.annotations }}
```

Comment:
this will fail CI because the crds are generated automatically from the ones under config. You should change inv so it injects this section

Reply:
I've added generation of these `{{` annotations in some as little-hacky-as-possible way 
Please re-check